bigml/ensemble.py

```diff
@@ -253,12 +280,63 @@ def list_models(self):
         """
         return self.model_ids
 
+    def predict_probability(self, input_data, method=PROBABILITY_CODE,
```

Comment:
Is there a reason to use the PROBABILITY_CODE as default? For the predict method in the ensemble we use PLURALITY as the default combiner. Is better to use the probability weighted combiner in this case?